### PR TITLE
Add support for conda-lock install

### LINKED ISSUE #43: Add command to perform a conda install
```
conda-lock install [-p {foo}|-n {foo}] mylockfile.lock --additional args to conda
```

### PR DESCRIPTION
**Pull request by munali**
Closes #43

### CODE REVIEW

File: tests/test_conda_lock.py [hunk 1]
```diff
@@ -177,3 +184,164 @@ def test_aggregate_lock_specs():
...
+    return True
+
+
+@pytest.fixture
+def conda_exe():
+    return determine_conda_executable("conda", no_mamba=True)
+
+
+@pytest.fixture
+def conda_env(conda_exe):
```
Comment (mariusvniekerk) [inline tests/test_conda_lock.py:272]: I don't think we need this fixture, since this tests that we can install things in a way we're not using?

Comment (mariusvniekerk) [reply tests/test_conda_lock.py:272]: we can use the destroy with a finally in the test

File: conda_lock/conda_lock.py [hunk 2]
```diff
@@ -121,6 +123,58 @@ def print_proc(proc):
...
+        if proc.stdout:
+            print(f"    STDOUT:\n{proc.stdout}")
+        if proc.stderr:
+            print(f"    STDERR:\n{proc.stderr}")
+
+    try:
+        proc.check_returncode()
+    except subprocess.CalledProcessError:
+        try:
+            err_json = json.loads(proc.stdout)
```
Comment (mariusvniekerk) [inline conda_lock/conda_lock.py:164]: this will only work if there is a `--json` as part of the conda create command

**Review by mariusvniekerk — APPROVED** [review]

File: .travis.yml [hunk 3]
```diff
@@ -17,10 +17,6 @@ matrix:
       env: PY=3.7
     - name: "python-3.8"
       env: PY=3.8
-    - name: "coding_standards"
-      env: PY=3
-    - name: "tarball"
```
Comment (ocefpaf) [inline .travis.yml:22]: Why remove these tests?

File: .travis.yml [hunk 4]
```diff
@@ -17,10 +17,6 @@ matrix:
       env: PY=3.7
     - name: "python-3.8"
       env: PY=3.8
-    - name: "coding_standards"
```
Comment (ocefpaf) [inline .travis.yml:20]: This one can go now that we have pre-commit hooks that do the same.

**Review by ocefpaf — APPROVED** [review]

File: .travis.yml (hunk 3 above)
Comment (ocefpaf) [reply .travis.yml:22]: The tarball test do help prevent bad releases. I recommend to add them back.

**Pull request merged**